draft-cutler-httpbis-partitioned-cookies.md

```diff
@@ -104,6 +104,18 @@ The following could also be added as an additional step to section 5.4:
       If the cookie-attribute-list does contain an attribute with an attribute-name of "Partitioned" and the secure-only-flag is false, abort these steps and ignore the cookie entirely.<br><br>
       Otherwise, set partition-key to cookie-partition-key defined in section 5.X.X.
 
+## Partitioned Cookies with the Same Name/Domain/Path
+
+In order to prevent cross-partition leaks, we need to allow sites to set cookies with the same name, domain, and path as a cookie with another partition key.
```

Comment:
```suggestion
In order to prevent cross-partition leaks, we need to allow sites to set cookies with the same name, domain, and path as long as they have different partition keys.
```